src/internal/pgjsontypes/pgjsontypes.go

```diff
@@ -8,20 +9,20 @@ import (
 	"github.com/pachyderm/pachyderm/v2/src/internal/errors"
 )
 
-// jsonMap wraps a map[string]string that is stored as JSONB in the database.  While structs can be
+// StringMap wraps a map[string]string that is stored as JSONB in the database.  While structs can be
```

Comment:
Future expansion to other types is not planned.  We use JSONB because everyone at the design review preferred that to a table with FK -> (key, value).